### PR TITLE
Feature/form styling

### PR DESCRIPTION
**Pull request by SeanPed**

### CODE REVIEW

File: src/app/components/AddButton/AddButton.module.css [hunk 1]
```diff
@@ -1,17 +1,17 @@
 .add_button {
-  background-color: hotpink;
+  background-color: cornflowerblue;
```
Comment (derjubu) [inline src/app/components/AddButton/AddButton.module.css:2]: Why did you edit this in two pull requests? 🧐

File: src/app/components/Form/Form.module.css [hunk 2]
```diff
@@ -1,19 +1,64 @@
...
+}
+
+.submit_button {
+  background-color: #08d;
+  border-radius: 12px;
+  border: 0;
+  box-sizing: border-box;
+  color: #eee;
+  cursor: pointer;
+  font-size: 18px;
```
Comment (derjubu) [inline src/app/components/Form/Form.module.css:20]: You should use rem or em

File: src/app/components/Form/Form.module.css [hunk 3]
```diff
@@ -1,19 +1,64 @@
...
   font-size: 90px;
 }
+.lable_weight {
+  color: grey;
+}
+.input_field {
+  background-color: #303245;
+  border-radius: 12px;
+  border: 0;
+  box-sizing: border-box;
```
Comment (derjubu) [inline src/app/components/Form/Form.module.css:42]: You could put this in your global css

File: src/app/components/Form/Form.module.css [hunk 4]
```diff
@@ -1,19 +1,64 @@
...
+  border-radius: 12px;
+  border: 0;
+  align-self: center;
+  color: #eee;
+  font-size: 18px;
+  outline: 0;
+  padding: 2px 10px 0;
+  width: 30%;
+}
+.spacing_div {
```
Comment (derjubu) [inline src/app/components/Form/Form.module.css:62]: This looks like it should be part of components and not a separate element ?🧐

File: src/app/components/Form/Form.tsx [hunk 5]
```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
...
-      </label>
-      <label>
-        Gewicht
-        <br />
-        <br />
-        <input type="number" defaultValue={0.5} min={0.5} max={25} step={0.5} />
-        <br />
-      </label>
+    <form className={classes.input_form} action="">
       <br />
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:7]: Why do you need <br />? Can't you do this with styling?

File: src/app/components/Form/Form.tsx [hunk 6]
```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
...
-      <br />
-      <input className={classes.inputButton} type="submit" value="Submit" />
+      <input
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:8]: Where is you label? 👀

File: src/app/components/Form/Form.tsx [hunk 7]
```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
...
+        className={classes.input_field}
+        type="text"
+        autoComplete="off"
+        placeholder="Name des Gegenstandes"
+      />
+      <div className={classes.spacing_div} />
+      <textarea
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:15]: This label is also missing 🔍

File: src/app/components/Form/Form.tsx [hunk 8]
```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
...
+        className={classes.input_field}
+        rows={5}
+        placeholder="Beschreibung des Gegenstandes"
+      ></textarea>
+      <div className={classes.spacing_div} />
+      <div className={classes.spacing_div} />
+      <label className={classes.lable_weight}>
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:22]: This label is not connected to an input field 😿

File: src/app/components/Form/Form.tsx [hunk 9]
```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
...
+        Gewicht in Kilogramm <br />
+        (0,2kg Schritte!)
+      </label>
+      <div className={classes.spacing_div} />
+      <input
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:27]: Also no label 🕵️

**Review by derjubu — CHANGES_REQUESTED** [review]
You need to look at your labels/inputs once again.
Apart from that: Good job 👍

File: src/app/components/Form/Form.module.css [hunk 10]
```diff
@@ -1,19 +1,64 @@
...
-  border-left: none;
-  border-right: none;
-  background: linear-gradient(#5fddff, #53addf);
-  color: white;
-  border-radius: 10px;
-  box-shadow: 0 2px 10px grey;
-  transition: 150ms ease;
-  font-weight: bold;
-  margin-top: 10px;
+textarea {
```
Comment (PReichetanz) [inline src/app/components/Form/Form.module.css:1]: If you use className almost everywhere, I would suggest to use it here as well.

File: src/app/components/Form/Form.tsx (hunk 6 above)
Comment (PReichetanz) [inline src/app/components/Form/Form.tsx:8]:
```suggestion
     <label htmlFor="name">Name des Gegenstandes</label>
       <input
       id="name"
```

File: src/app/components/Form/Form.tsx [hunk 11]
```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
...
+      <textarea
+        className={classes.input_field}
+        rows={5}
+        placeholder="Beschreibung des Gegenstandes"
+      ></textarea>
+      <div className={classes.spacing_div} />
+      <div className={classes.spacing_div} />
+      <label className={classes.lable_weight}>
+        Gewicht in Kilogramm <br />
+        (0,2kg Schritte!)
```
Comment (PReichetanz) [inline src/app/components/Form/Form.tsx:24]: I suggest to use a span element here with display:block; this way, you can style the comment if you want and you don't rely on <br/>
```suggestion
        <span>(0,2kg Schritte!)</span>
```

**Review by PReichetanz — COMMENTED** [review]
Nice work, I only added minor thing which came to my mind :)

**Pull request merged**